src/components/nav.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import '../modules-css/nav.css';
+
+const Header = () => (
+  <header className="header">
+    <h1 className="logo">Math Magicians</h1>
+    <div>
+      <ul className="ul-nav">
+        <li className="link-item">
+          <NavLink to="/">
+            Home
+          </NavLink>
+        </li>
+        <li className="link-item">
+          <NavLink to="/calculator">
+            About
+          </NavLink>
```

Comment:
- [ ] Great job 👏. This page should be called `Calculator` not `About` as seen in the project requirements 🙏